deployments/zksync.json

```diff
@@ -23,7 +23,7 @@
   "AcrossFacetPackedV3": "0x45C83B209878601d21604301dfE25bD6D0BBC218",
   "GasZipFacet": "0x313c27Aad40c7e0A0b923b539F05617D8114566D",
   "GasZipPeriphery": "0x4000f76450d9d40558fd60F66863F583B7A5BCCb",
-  "Permit2Proxy": "0x6275f6631c955DC5dA9fBe8Dc7f24a3A5919443A",
+  "Permit2Proxy": "0x2ec2c4C3dc212C990d1bc2b48b0392A3951D926E",
```

Comment:
in networks.json the settings are london and 0.8.17 but deployed cancun and 0.8.26 but it's zksync so do we handle it differently?

Reply:
yes, zkEVM networks will automatically always deploy with 0.8.26 and zkEVM solc 1.5.3 

from foundry.toml:
<img width="538" height="169" alt="image" src="https://github.com/user-attachments/assets/71719442-c9fa-4212-8bec-24cf35b99d07" />

Reply:
ok, forgot that we have that profile for zksync